hedera-mirror-rest/transactions.js

```diff
@@ -777,17 +784,51 @@ const extractSqlFromTransactionsByIdOrHashRequest = async (transactionIdOrHash,
   }
 
   mainConditions.unshift(...commonConditions);
-  return {query: getTransactionQuery(mainConditions.join(' and '), commonConditions.join(' and ')), params};
+  return {
+    query: getTransactionQuery(mainConditions.join(' and '), commonConditions.join(' and ')),
+    params,
+    scheduledParamExists: scheduledParamExists,
+    isTransactionHash: isTransactionHash,
+  };
 };
 
+function getTransactionsByIdOrHashCacheControlHeader(transactionsRows, isTransactionHash, scheduled) {
```

Comment:
coding style consistency, please use arrow function

```suggestion
const getTransactionsByIdOrHashCacheControlHeader = (transactionsRows, isTransactionHash, scheduled) =>  {
```